src/broadcast/spectateManager.ts

```diff
@@ -30,14 +30,15 @@ const generatePlaybackId = (broadcastId: string) => `spectate-${broadcastId}`;
  */
 export class SpectateManager extends EventEmitter {
   private broadcastInfo: Record<string, BroadcastInfo> = {};
+  private broadcastList: Record<string, BroadcasterItem> = {};
```

Comment:
These two variables are too similar in name. It's really not clear what contains what. I think we should rename them. Maybe something like this?
```suggestion
  private currentlySpectating: Record<string, BroadcastInfo> = {};
  private availableBroadcasts: Record<string, BroadcasterItem> = {};
```
Not sure on the name of `currentlySpectating` but something like `activeBroadcasts` isn't that much more descriptive, cuz technically an "available" broadcast is also an active one.